OpenNMT/onmt/Dataset.py

```diff
@@ -46,3 +50,9 @@ def __getitem__(self, index):
 
     def __len__(self):
         return self.numBatches
+
+
+    def shuffle(self):
```

Comment:
Yeah, I left it in as an option because I saw somewhat better scores using the completely shuffled data, though it does slow things down quite a bit. I can remove it.